Dockerfile

```
@@ -1,19 +1,17 @@
-FROM amazonlinux:2.0.20180622.1
+FROM python:3-alpine
 LABEL maintainer="Rackspace"
 
-ENV APP_HOME /tuvok
+ENV PATH="/root/.local/bin:${PATH}"
+ENTRYPOINT [ "/bin/sh" ]
 
-# Install OS packages
-RUN yum update -y && \
-    yum install -y git python37 jq
+# Install packages/updates/dependencies
+ADD https://github.com/kvz/json2hcl/releases/download/v0.0.6/json2hcl_v0.0.6_linux_amd64 /usr/local/bin/json2hcl
+RUN chmod +x /usr/local/bin/json2hcl
+RUN apk --update add git openssh curl jq && pip3 install --upgrade pip
 
-# Install other tools (not packaged)
-RUN curl -SsL https://github.com/kvz/json2hcl/releases/download/v0.0.6/json2hcl_v0.0.6_linux_amd64 \
-  | tee /usr/local/bin/json2hcl > /dev/null && chmod 755 /usr/local/bin/json2hcl && json2hcl -version
+ADD . ${WORKDIR}
+RUN pip3 install --user -r test-requirements.txt && pip3 install --user -r requirements.txt && pip3 install --user -e .
 
-RUN mkdir -p ${APP_HOME}
-ADD . ${APP_HOME}
-
-RUN cd ${APP_HOME} && pip3 install --user -r test-requirements.txt -e .
-
-ENTRYPOINT [ "python3", "tuvok/tuvok/cli.py" ]
+# just try to run tuvok, so we can be sure it is installed correctly
+RUN tuvok --version
```

do we really need this line, since we have tests and all?

---

No, but I wanted it to fail faster than waiting for CircleCI to spin up a new container for tests.